jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -1191,11 +1208,13 @@ def _make_conv_harness(name, *, lhs_shape=(2, 3, 9, 10), rhs_shape=(3, 3, 4, 5),
     # TODO(bchetioui): the NCW data format is not supported on CPU for TF
     # for now. That path is thus disabled to allow the code to use XLA instead.
   ]
+  for enable_xla in [False, True]
```

Comment:
I like this as well: all tests either use the default enable_xla=True or try both cases.